src/components/UI/Card.js

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { VscEdit } from "react-icons/vsc";
+import { VscSave } from "react-icons/vsc";
+import { VscClose } from "react-icons/vsc";
+//import classnames from 'classnames';
+
+import "./Card.css";
+
+var classNames = require("classnames");
+
+class Card extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: false,
+      clicked: false,
+        input: "",
+        inputValue: "",
+        textarea: "",
+        textareaValue: "",
```

Comment:
Move those to separate objects:
data = {input: "", textarea: ""} - passed to Card via props.
tempData= {input:"", textArea: ""} - stored in component state.